mono/mini/driver.c

```diff
@@ -1689,6 +1689,18 @@ mono_get_version_info (void)
 
 static gboolean enable_debugging;
 
+void
+handle_stats_name (char *arg)
+{
+	size_t method_name_len = strlen (arg);
+	if (method_name_len == 0) {
```

Comment:
This will never be zero because you're comparing the full argv, including the `--stats=` portion. I recommend passing only the method name itself to this function.